multiplechildren.php

```diff
@@ -169,31 +171,14 @@ function multiplechildren_civicrm_validateForm($formName, &$fields, &$files, &$f
  */
 function multiplechildren_civicrm_buildForm($formName, &$form) {
   if ($formName == "CRM_Event_Form_ManageEvent_EventInfo") {
-    $templateId = NULL;
-    if (!empty($form->_id)) {
-      $template = civicrm_api3('Event', 'get', [
-        'id' => $form->_id,
-        'return.custom_' . MINISTRY => 1,
-      ])['values'][$form->_id];
-      if (!empty($template['custom_' . MINISTRY])) {
-        $templateId = $template['custom_' . MINISTRY];
-      }
-      $mulChild = new CRM_Multiplechildren_DAO_MultipleChildren();
-      $mulChild->event_id = $form->_id;
-      $mulChild->find(TRUE);
-      if ($mulChild->multiple_child) {
-        $form->setDefaults(['multiple_children' => 1]);
-      }
-      else {
-        $form->setDefaults(['multiple_children' => 0]);
-      }
-    }
     if (empty($form->getVar('_isTemplate'))) {
       // This is not a ministry reportable event, so we show the register multiple children checkbox.
       $form->addYesNo('multiple_children', ts('Register Multiple Children?'));
       CRM_Core_Region::instance('page-body')->add(array(
         'template' => 'CRM/MultipleChildren.tpl',
       ));
+      // Set this to default = TRUE.
+      $form->setDefaults(['multiple_children' => 1]);
```

Comment:
@Edzelopez I think that this should only occur if we are adding a new event, otherwise it will potentially cause confusion because the radio won't match what is currently saved in the database

Reply:
Good catch, fixed this now.